pkg/bloombuild/planner/queue/config.go

```diff
@@ -2,20 +2,31 @@ package queue
 
 import (
 	"flag"
+	"fmt"
 
 	"github.com/grafana/loki/v3/pkg/queue"
 )
 
 type Config struct {
-	MaxQueuedTasksPerTenant int `yaml:"max_queued_tasks_per_tenant"`
+	MaxQueuedTasksPerTenant int    `yaml:"max_queued_tasks_per_tenant"`
+	StoreTasksOnDisk        bool   `yaml:"store_tasks_on_disk"`
+	TasksDiskDirectory      string `yaml:"tasks_disk_directory"`
+	CleanTasksDirectory     bool   `yaml:"clean_tasks_directory"`
 }
 
 // RegisterFlagsWithPrefix registers flags for the bloom-planner configuration.
 func (cfg *Config) RegisterFlagsWithPrefix(prefix string, f *flag.FlagSet) {
 	f.IntVar(&cfg.MaxQueuedTasksPerTenant, prefix+".max-tasks-per-tenant", 30000, "Maximum number of tasks to queue per tenant.")
+	f.BoolVar(&cfg.StoreTasksOnDisk, prefix+".store-tasks-on-disk", false, "Whether to store tasks on disk.")
+	f.StringVar(&cfg.TasksDiskDirectory, prefix+".tasks-disk-directory", "/tmp/bloom-planner-queue", "Directory to store tasks on disk. All files will be removed on startup.")
```

Comment:
```suggestion
	f.StringVar(&cfg.TasksDiskDirectory, prefix+".tasks-disk-directory", "/tmp/bloom-planner-queue", "Directory to persist tasks on disk across restarts.")
```

Reply:
We are not yet doing this.

> `All files will be removed on startup`

Removing this bit.